recipe/meta.yaml

```diff
@@ -1,46 +1,53 @@
 {% set name = "dash" %}
 {% set version = "2.18.2" %}
-{% set sha256 = "20e8404f73d0fe88ce2eae33c25bbc513cbe52f30d23a401fa5f24dbb44296c8" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/dash-{{ version }}.tar.gz
+  sha256: 20e8404f73d0fe88ce2eae33c25bbc513cbe52f30d23a401fa5f24dbb44296c8
 
 build:
-  number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  entry_points:
+    - dash-generate-components = dash.development.component_generator:cli
+    - renderer = dash.development.build_process:renderer
+    - dash-update-components = dash.development.update_components:cli
   noarch: python
+  script:
+    - sed -i.bak "s/Flask>=1.0.4,<3.1/flask>=1.0.4/g" requirements/install.txt
+    - sed -i.bak "s/Werkzeug<3.1/werkzeug/g" requirements/install.txt
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 1
 
 requirements:
+  build:
+    - sed     # [unix]
+    - m2-sed  # [win]
   host:
-    - python >=3.6
-    - setuptools
+    - python {{ python_min }}
     - pip
-  run:
-    - python >=3.6
     - setuptools
+  run:
+    - python >={{ python_min }}
+    # https://github.com/plotly/dash/blob/v2.18.2/requirements/install.txt
     - flask >=1.0.4
     - werkzeug
     - plotly >=5.0.0
-    - typing-extensions >=4.1.1
+    - dash-html-components >=2.0.0
+    - dash-core-components >=2.0.0
+    - dash-table >=5.0.0
```

Comment:
There were all in run constrained because dash moved them from separate repos into the dash code base. If we have them in run, then it may try to install the old outdated standalone packages, which don't exist. Actually, I'm unsure why this recipe built if they are present here because I don't think these three packages versions even exist.

Reply:
These versions do seem to exist on conda forge, but I still don't think we want them installed. We just want them to be at a high enough version if they happen to be in the environment dash is installed in, in case it is imported from the standalone package and not the copy maintained in dash.

Reply:
Also, the two test files where there to likely check that the standalone packages were not installed.

Reply:
Can you revert all the details regarding html-components, core-components, and table. They were functioning as expected.